pennylane/qchem/convert.py

```diff
@@ -303,7 +303,8 @@ def _openfermion_pennylane_equivalent(
 def import_operator(qubit_observable, format="openfermion", wires=None, tol=1e010):
     r"""Convert an external operator to a PennyLane operator.
 
-    The external format currently supported is openfermion.
+    The external format currently supported is openfermion: the function takes most types of
+    OpenFermion qubit operators, such as those corresponding to Pauli words and summations thereof.
```

Comment:
Not a mandatory change, just if you think it improves clarity. Maybe this actually sounds worse :laughing: 
```suggestion
    We currently support [openfermion](https://quantumai.google/openfermion) operators: the function accepts most types of
    OpenFermion qubit operators, such as those corresponding to Pauli words and sums of Pauli words.
```

Reply:
I like it! I'll take active voice over passive any day haha